ui/components/app/security-provider-banner-alert/blockaid-banner-alert/blockaid-banner-alert.js

```diff
@@ -39,12 +39,19 @@ const REASON_TO_DESCRIPTION_TKEY = Object.freeze({
 /** List of suspicious reason(s). Other reasons will be deemed as deceptive. */
 const SUSPCIOUS_REASON = [BlockaidReason.rawSignatureFarming];
 
-function BlockaidBannerAlert({
-  ppomResponse: { reason, resultType, features },
-}) {
+function BlockaidBannerAlert({ securityAlertResponse }) {
   const t = useContext(I18nContext);
 
-  if (resultType === BlockaidResultType.Benign) {
+  if (!securityAlertResponse) {
+    return null;
+  }
+
+  const { reason, result_type: resultType, features } = securityAlertResponse;
+
+  if (
+    resultType === BlockaidResultType.Benign ||
+    resultType === BlockaidResultType.Failed
+  ) {
```

Comment:
According to our discussion @jpuri we are supposed to return the new banner alert when result type is Failed.

Reply:
@segun : UI changed for `failed` status will be implemented in a following PR.

Reply:
could be helpful to add in the description. working on it here:
https://github.com/MetaMask/MetaMask-planning/issues/1056